delivery_carrier_seur/model/stock.py

```diff
@@ -0,0 +1,160 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 FactorLibre (http://www.factorlibre.com)
+#                  Ismael Calvo <ismael.calvo@factorlibre.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields, api, exceptions
+from openerp.tools.translate import _
+from seur.picking import Picking
+from unidecode import unidecode
+from urllib2 import HTTPError
+
+
+class ShippingLabel(models.Model):
+    _inherit = 'shipping.label'
+
+    @api.model
+    def _get_file_type_selection(self):
+        """ To inherit to add file type """
+        res = super(ShippingLabel, self)._get_file_type_selection()
+        res.append(('txt', 'TXT'))
```

Comment:
If you have 2 carriers which use 'txt', then your field have a duplicate value ?
set can used to avoid it
https://github.com/OCA/carrier-delivery/blob/7.0/delivery_carrier_label_gls/stock.py#L345